### PR TITLE
docker-sync

### PR DESCRIPTION
**Pull request by arosenhagen**
implement [docker-sync](https://github.com/EugenMayer/docker-sync)
use with unison to really speed up things on macOS

### CODE REVIEW

**Comment by bvogel** [comment]
I really like the idea behind `docker-sync`, but this should be optional.

**Comment by arosenhagen** [comment]
the only way to make this optional is to have a separate `docker-compose.yml`

**Comment by shyim** [comment]
This project uses psh. Whats about a command which generates a docker-compose.override.yml for the mac users?

**Comment by arosenhagen** [comment]
I like the idea of `docker-compose.override.yml`(now that I read about it) - but I don't know how to integrate it cleanly. There has to be a setting on which one can turn on/off docker-sync - either as parameter to psh or in .psh.yaml, right?

**Review by dhwlz — APPROVED** [review]

File: docker-sync.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+version: "2"
+
+syncs:
+  #IMPORTANT: ensure this name is unique and does not match your other application container name
+  app_server-sync: #tip: add -sync and you keep consistent names as a convention
```
Comment (PascalThesing) [inline docker-sync.yml:5]: Hey, please add the right `sync_userid` and `sync_groupid`. Also, exclude the unnecessary files e.g. 
`sync_excludes: ['.git', '.idea', 'dev-ops/docker/_volumes', 'var', '.docker-sync']`.

Comment (PascalThesing) [reply docker-sync.yml:5]: You can see an example here: https://github.com/shopware/development/commit/d161be8abd32a348cfdb0d3aaa9831738013d93f

**Pull request closed without merging**